libs/coin-polkadot/src/hw-getAddress.ts

```diff
@@ -1,13 +1,16 @@
-import Polkadot from "@ledgerhq/hw-app-polkadot";
-import type { Resolver } from "@ledgerhq/coin-framework/bridge/getAddressWrapper";
+import { GetAddressFn } from "@ledgerhq/coin-framework/bridge/getAddressWrapper";
```

Comment:
With these changes, is this file / function still coin (polkadot) specific or could it be moved in coin framework?
Maybe we could use a Typescript Generic type for the `signerFactory` object that still seems to be coin specific?

Reply:
You're right about this one.
We can take a look at this in another ticket/branch :)